composer.json

```diff
@@ -90,7 +90,7 @@
     },
     "config": {
         "platform": {
-            "php": "8.1.0"
```

Comment:
Why is this change needed? Seems like `shish/safe` supports PHP 8.1 

https://github.com/shish/safe/blob/98133860236ca2308a0811d52b957c2a7807b710/composer.json#L105

Reply:
Not in the released version unfortunately 

https://packagist.org/packages/shish/safe